packages/updating-element/README.md

```diff
@@ -0,0 +1,174 @@
+# LitElement 3.0 Pre-release
```

Comment:
Update this? It can be extremely minimal for now.

Reply:
Done.